Missing change from upstream kernel commit
43598813386f6205edf3c21f1fe97f731ccb4f15 introduced by Cumulus commit
a9f6f3172cb6384dc7dec7b2aeb046e75577cb26. This patch should be removed
to complete CM-3094 and the original upstream patch should be added to
the patch stack.

diff --git a/net/bridge/br_forward.c b/net/bridge/br_forward.c
index 66a6f25..5b28ba9 100644
--- a/net/bridge/br_forward.c
+++ b/net/bridge/br_forward.c
@@ -124,7 +124,7 @@ static void __br_forward(const struct net_bridge_port *to, struct sk_buff *skb)
 /* called with rcu_read_lock */
 void br_deliver(const struct net_bridge_port *to, struct sk_buff *skb)
 {
-	if (should_deliver(to, skb)) {
+	if (to && should_deliver(to, skb)) {
 		__br_deliver(to, skb);
 		return;
 	}
